neo4j/test-integration/examples_test.go

```diff
@@ -498,6 +498,44 @@ func addPersonInTxFunc(driver neo4j.Driver, name string) error {
 
 // end::transaction-function[]
 
+// tag::transaction-timeout-config[]
+func configTxTimeout(driver neo4j.Driver, name string) error {
+	session := driver.NewSession(neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
```

Comment:
if you want to keep the example concise, you can remove the default `AccessMode` attribute here and below